.devcontainer/web/devcontainer.json

```diff
@@ -0,0 +1,33 @@
+// This file describes a development container, e.g., for use in an IDE.
```

Comment:
Good question. Based on the number of open issues I've found relating to personal overrides, I don't think there's a great way to address that with dev containers right now:
- https://github.com/microsoft/vscode-remote-release/issues/3279
- https://github.com/microsoft/vscode-remote-release/issues/3218
- https://github.com/microsoft/vscode-remote-release/issues/7650
- https://github.com/devcontainers/spec/issues/305

I think the main thing people would want to override is extensions, as you mention. For people using VS Code, the next best thing to git-ignored personal extensions for dev containers is: 
- there is a UserSetting for `Dev Containers: Default Extensions`. I've moved the Github Copilot extension there.
    - This isn't perfect, because it applies to all dev containers, and that's not appropriate for all extensions. But it is for copilot, so, cool.
- You can install extensions in the container after it's built, but then they're rather more temporary. 
 
I would mildly prefer to commit this file, in case it's useful to someone else, though I'm also happy to have it git-ignored.

Reply:
I don't think it will hurt, seems like worst that might happen is someone adding extra extensions/config stuff that might not be applicable to everyone and will probably just be ignored with maybe some error messages.

Reply:
Though on second thought if someone makes many personal customizations to it, they may have to deal with conflicts and rebases to this file quite frequently.   But I'm okay with adding it for now, it can be removed later if it does cause problems.